agir/front/components/app/Navigation/TopBar/MobileNavBar/RightLink.js

```diff
@@ -1,17 +1,49 @@
 import PropTypes from "prop-types";
 import React, { useState } from "react";
+import styled from "styled-components";
 
 import Avatar from "@agir/front/genericComponents/Avatar";
 import BottomSheet from "@agir/front/genericComponents/BottomSheet";
 import { RawFeatherIcon } from "@agir/front/genericComponents/FeatherIcon";
 import Spacer from "@agir/front/genericComponents/Spacer";
+import style from "@agir/front/genericComponents/_variables.scss";
 
 import { IconLink } from "./StyledBar";
 import UserMenu from "../UserMenu";
+import { useRouteMatch } from "react-router-dom";
+import useSWR from "swr";
+import { useToast } from "@agir/front/globalContext/hooks";
+import { switchMessageMuted, getGroupEndpoint } from "@agir/groups/api.js";
+
+const BlockMuteMessage = styled.div`
+  ${({ isActive }) => !isActive && `color: red;`}
+  ${RawFeatherIcon}:hover {
+    color: ${style.primary500};
+  }
+`;
 
 export const RightLink = (props) => {
   const { isLoading, user, settingsLink } = props;
   const [isUserMenuOpen, setIsUserMenuOpen] = useState(false);
+  const sendToast = useToast();
+  const matchMessagePage = useRouteMatch("/messages/:messagePk/");
+  const messagePk = matchMessagePage?.params.messagePk;
+  const { data, mutate } = useSWR(
+    getGroupEndpoint("getMessageMuted", { messagePk })
```

Comment:
Pour éviter de faire des calls API pour rien
```suggestion
    messagePk && getGroupEndpoint("getMessageMuted", { messagePk })
```